lib/find-view.js

```diff
@@ -235,7 +235,7 @@ class FindView {
       'find-and-replace:use-selection-as-replace-pattern': this.setSelectionAsReplacePattern.bind(this)
     }));
 
-    this.refs.replaceNextButton.addEventListener('click', e => e.shiftKey ? this.replacePrevious() : this.replaceNext()));
+    this.refs.replaceNextButton.addEventListener('click', e => e.shiftKey ? this.replacePrevious.bind(this) : this.replaceNext.bind(this));
```

Comment:
@icecream17 I don't think this is correct. Because `e => this.replacePrevious() : this.replaceNext()` is a function.